client/nginx.conf.docker

```diff
@@ -19,6 +19,9 @@ http {
     server_tokens off;
     keepalive_timeout 65;
 
+    proxy_cache_path /tmp/nginx-cache
+        levels=1:2 keys_zone=spa_cache:1m max_size=50m inactive=60m use_temp_path=off;
```

Comment:
This is the part here where I wanted to try to set these values via environment variables.

No clue if it's easy or not, but I'll at least try to look around.

Reply:
We could just do a sed substitution like we do for the backend hostname.

Reply:
Since we're only caching the HTML stub pages, and each stub page is at most 3.5 kB, the 50 MB cache size would be able to store about 14,000 stub pages for about 60 minutes each. I don't really see this happening for the majority of users, and those that would see this will probably fork the project anyway to make more involved source code modifications.